cli/mizu/config.go

```diff
@@ -95,7 +105,7 @@ func initFlag(f *pflag.Flag) {
 
 	if f.Name == SetCommandName {
 		if setError := mergeSetFlag(sliceValue.GetSlice()); setError != nil {
-			Log.Infof(uiUtils.Red, "Invalid set argument")
+			Log.Warningf(uiUtils.Red, fmt.Sprintf("Invalid set argument, error: %v", setError))
```

Comment:
we can print here the available options

Reply:
allowedSetFlags

Reply:
The log will look like:
`Invalid set argument, error: invalid set key AAA`
Can we remove the duplication, make more user friendly